setup-scripts/skynet-nginx.conf

```diff
@@ -17,12 +17,13 @@ server {
 	}
 
 	location /skynet/skyfile/ {
+		add_header Access-Control-Allow-Origin: *;
```

Comment:
maybe group both CORS changes and document it a little, no need to go into it at length though I think